### PR TITLE
Some weapon property clarifications

### PR DESCRIPTION
**Pull request by warp-core**
**Correction/Clarification**

- From: https://github.com/endless-sky/endless-sky/issues/9692

## Summary
Clarifies the details mentioned in the linked issue:

- Include a brief description on when a weapon will be considered "continuous" in the CreatingOutfits page, near where reload, burst count, and burst reload are mentioned, as relevant properties.
  - Also requested in https://github.com/endless-sky/endless-sky/issues/9536
- Mention that weapons with blast radius will not damage or prospect minables.
- Mention that "disabled damage" does not override "relative hull damage".
- Mention that "relative minable damage" can be non-zero if undefined and "relative hull damage" is provided.

### CODE REVIEW

File: wiki/CreatingOutfits.md [hunk 1]
```diff
@@ -792,7 +800,7 @@ Ordinary weapon attributes (those that take a number as an argument) include:
 
 * `"trigger radius"`: how close a projectile must be to a hostile target to trigger its explosion. This only makes sense to use with weapons with a `"blast radius"` at least as big as their `"trigger radius"`.
 
-* `"blast radius"`: all ships (friendly and hostile) within this radius are damaged if this projectile explodes. Note: anything with a blast radius will show up on radar, like missiles do. Beginning in **v. 0.9.9**, any weapon with a blast radius is subject to damage scaling - objects closer to the center of the blast take more damage. Weapons with a trigger radius have their nominal damage boosted a small amount to compensate.
+* `"blast radius"`: all ships (friendly and hostile) within this radius are damaged if this projectile explodes. Note: anything with a blast radius will show up on radar, like missiles do. Beginning in **v. 0.9.9**, any weapon with a blast radius is subject to damage scaling - objects closer to the center of the blast take more damage. Weapons with a trigger radius have their nominal damage boosted a small amount to compensate. Weapons with a blast radius will not deal damage or apply prospecting to a minable.
```
Comment (TomGoodIdea) [inline wiki/CreatingOutfits.md:803]: does it mean we're not going to fix it?

Comment (warp-core) [reply wiki/CreatingOutfits.md:803]: I'm just making the documentation more complete and accurate.

**Comment by TheGiraffe3** [comment]
@warp-core did you see my comment?

**Review by TheGiraffe3 — APPROVED** [review]

File: wiki/CreatingOutfits.md [hunk 2]
```diff
@@ -766,10 +766,18 @@ Ordinary weapon attributes (those that take a number as an argument) include:
 
 * `reload`: how many frames this weapon takes to reload: 1 means it fires every turn (e.g. most beam weapons), and 60 means it fires once per second.
 
+  * A non-burst weapon with a reload of 1 or less and a total lifetime of 1 will have a fire rate of "continuous" in its outfit info.
+
+  * The total lifetime is the lifetime of the initial project added to the total lifetime of its longest lived submunition, calculated recursively, accounting for the submunitions of the submunition, etc.
```
[the remark this answers is not in the document]
Comment (warp-core) [reply wiki/CreatingOutfits.md:771]:
```suggestion
  * The total lifetime is the lifetime of the initial projectile added to the total lifetime of its longest lived submunition, calculated recursively, accounting for the submunitions of the submunition, etc.
```

**Review by tibetiroka — APPROVED** [review]

**Pull request merged**